tpl/tplimpl/template_embedded.go

```diff
@@ -84,7 +84,7 @@ func (t *GoHTMLTemplate) EmbedTemplates() {
       <pubDate>{{ .Date.Format "Mon, 02 Jan 2006 15:04:05 -0700" | safeHTML }}</pubDate>
       {{ with .Site.Author.email }}<author>{{.}}{{ with $.Site.Author.name }} ({{.}}){{end}}</author>{{end}}
       <guid>{{ .Permalink }}</guid>
-      <description>{{ .Content | html }}</description>
+      <description>{{ .Summary | html }}</description>
```

Comment:
Even if the Atom spec allows the usage of the full content I think the original intention of RSS is to deliver a sneak-peak for content I subscribed to. Hence the usage of `.Summary` as replacement makes sense. Many themes just show the summary on the homepage as well.